client/src/components/relatedProducts/card.jsx

```diff
@@ -1,62 +1,118 @@
 /* eslint-disable no-unused-vars */
 import React from 'react';
-import axios from 'axios';
 import styled from 'styled-components';
 import TestStarIcon from '../stars/star.jsx'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
-import {faStar} from '@fortawesome/free-solid-svg-icons'
+import { faStar } from '@fortawesome/free-solid-svg-icons'
 
 const Carta = styled.div`
-border: solid;
-border-width: thin;
-border-color: #f2f2f2;
-width: 200px;
-height: 299px;
-`
-
-
-
-const Pic = styled.img`
-width: 200px;
-height: 195px;
+  border: solid 1px;
+  border-width: thin;
+  border-color: lightgrey;
+  width: 200px;
+  height: 299px;
 `
-const PicContainer=styled.div`
+const PicContainer = styled.div`
   width: 200px;
   height: 195px;
+  position: relative;
 `
+const StarBtn = styled.div`
+  float: right;
+  position: absolute;
+  right: 10px;
+  top: 2px;
+  color: cornflowerblue;
 
+`
+const Pic = styled.img`
+  width: 199px;
+  height: 195px;
+`
 const Container = styled.div`
- text-align: left;
- padding: 2px 16px;
- position: relative;
- bottom: 0;
+  text-align: left;
+  padding: 2px 16px;
+  position: relative;
+  bottom: 0;
 `
-
-
-const CardParagraphs = styled.p`
+const CategoryP = styled.p`
+  font-size: 11px;
+`
+const SaleP = styled.p`
+  font-size: 11px;
+  color: red;
+`
+const PriceP2 = styled.p`
   font-size: 11px;
+  text-decoration: line-through;
+`
+const PriceP = styled.p`
+font-size: 11px;
```

Comment:
It looks like you have a good hold on styled components. One thing that I can see is for a file of this size, you might want to break up the components into multiple files for the sake of things looking a little cleaner. I have to do the same.